web/modules/custom/ai_screening/src/Helper/UserHelper.php

```diff
@@ -48,12 +73,23 @@ public function setDefaultLanguage(UserInterface $user): UserInterface {
     return $user;
   }
 
+  /**
+   * Event handler for altering the username on display.
+   */
+  public function alterUserName(UserFormatNameAlterEvent $event): void {
+    $name = &$event->getName();
+    $user = $this->userStorage->load($event->getAccount()->id());
+    $fieldName = $user?->field_name->value;
```

Comment:
We usually use the `get` method to get a field (rather than a magic getter):

```suggestion
    $personName = $user?->get('field_name')->getString();
```